src/Moderator.cpp

```diff
@@ -69,19 +69,23 @@ unsigned Moderator::get_search_depth(const DomineeringState& state) const {
     unsigned game_moves = state.getNumMoves();
     unsigned depth;
 
+    if (searcher.get_time_left() > 20.0) {
```

Comment:
I pushed those changes! Why is 42 okay?

-----Original Message-----
From: "Naoki Mizuno" <notifications@github.com>
Sent: ‎11/‎8/‎2015 11:14 PM
To: "NigoroJr/uccineering" <uccineering@noreply.github.com>
Cc: "kojsmn" <kojsmn@miamioh.edu>
Subject: Re: [uccineering] Timer rush (#20)

In src/Moderator.cpp:
> @@ -69,19 +69,23 @@ unsigned Moderator::get_search_depth(const DomineeringState& state) const {
>      unsigned game_moves = state.getNumMoves();
>      unsigned depth;
>  
> +    if (searcher.get_time_left() > 20.0) {
The value looks legitimate. Just create a static constexpr float TIME_LIMIT = 20; or something. Arbitrary number (other than 42) appearing in code almost never looks good.
—
Reply to this email directly or view it on GitHub.